src/components/switch_language/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,35 @@
+import { useCallback, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import SwitchLanguageButton from "./SwitchLanguageButton";
+
+const LanguageSwitcher = () => {
+    const [activeLanguage, setActiveLanguage] = useState('ru');
```

Comment:
тут классно подойдет использование union type из typeScript 
```suggestion
type Language = 'ru' | 'en'


    const [activeLanguage, setActiveLanguage] = useState<Language>('ru');
```

тогда в activeLanguage не попадет ничего кроме 'ru' или 'en'